Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md

```diff
@@ -1,35 +1,43 @@
 ### Generate an API Key and API Key ID
 <~XSIAM>
-1. In your Cortex XSIAM platform, go to Settings -> Configurations -> API Keys.
+1. Go to Settings -> Configurations -> API Keys.
 2. Click the **+New Key** button in the top right corner.
 3. Generate a key of type **Advanced**.
 4. Copy and paste the key.
 5. From the ID column, copy the Key ID.
-
-### URL
-1. In your Cortex XSIAM platform, go to **Settings** -> API Keys.
-2. Click the **Copy URL** button in the top right corner.
 </~XSIAM>
 
-<~XSOAR>
-
-In version 8.0.0 and above:
+<~XSOAR_SAAS>
 1. Go to Settings -> API Keys.
```

Comment:
**Settings & Info** > **Settings** > **API Keys**.

Reply:
Should be the same as https://github.com/demisto/content/pull/31930/files#diff-19e1bedf2f2bb5d3de2b4f11e5f1fcbc8b532f8712c4f9add1ad0c4faef6b787R32